Assets/Scripts/Game/Items/ItemBuilder.cs

```diff
@@ -357,27 +354,45 @@ public static DaggerfallUnityItem CreateRandomArmor(int playerLevel, Genders gen
             int groupIndex = UnityEngine.Random.Range(0, enumArray.Length);
             DaggerfallUnityItem newItem = new DaggerfallUnityItem(ItemGroups.Armor, groupIndex);
 
-            // Adjust for gender
-            if (gender == Genders.Female)
```

Comment:
I notice this has been removed and PlayerTextureArchive is no longer being set by race and gender any more. Can you let me know where this is being done now? Cheers.